### DISCUSSION TITLE
Too much wasted white space

### DISCUSSION THREAD

**Discussion by pitosalas** [squidfunk/mkdocs-material#7005] [General]
Note nice display when browser is narrow:

<img width="862" alt="Screenshot 2024-04-01 at 2 23 08 PM" src="https://github.com/squidfunk/mkdocs-material/assets/44681/0d427501-62d1-4f1d-a90a-5650e2da78b1">

But now, I make it wider, and note plenty of white space on the left, and yet no navigation yet.

<img width="1165" alt="Screenshot 2024-04-01 at 2 23 16 PM" src="https://github.com/squidfunk/mkdocs-material/assets/44681/23be78a5-d40c-4103-b07c-cb4727e2b6ef">

And finally, with the window super wide, the navigation finally appears but there is still a ton of wasted white space on the left.

<img width="1284" alt="Screenshot 2024-04-01 at 2 23 28 PM" src="https://github.com/squidfunk/mkdocs-material/assets/44681/b2bcc5a9-1055-4720-a4a5-9628487dd70b">

I am not sure why you assume such wide windows. It would be great if I could control the thresholds when  you decide to display the navigation (either left hand-side or tabs which I think use the same thresholds.)

**Comment by zhyatt** [comment]
Perhaps this is the approach you'd like to take to allow the content width to grow without limiting it: https://github.com/squidfunk/mkdocs-material/discussions/6404#discussioncomment-7695243

**Reply by squidfunk** [reply]
[It's even documented](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#content-area-width) ☺️

**Reply by squidfunk** [reply]
On a second note, the screenshots in the OP look weird, as the header is kind of shrunk. Normally it looks like this:

![screenshot-squidfunk-github-io-mkdocs-material-getting-started-1712015248891](https://github.com/squidfunk/mkdocs-material/assets/932156/363668ac-aff7-4af7-9d9f-e242084f1c91)
